### PR TITLE
Final tested esp codes for the modules

### PR DESCRIPTION
**Pull request by Kavindya-Kariyawasam**
Testing codes for seperate modules we used in our project and some of the initial integrated tests with few modules.

### CODE REVIEW

File: CameraWebServer/CameraWebServer.ino [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+//#define CAMERA_MODEL_ESP32S2_CAM_BOARD
+//#define CAMERA_MODEL_ESP32S3_CAM_LCD
+//#define CAMERA_MODEL_DFRobot_FireBeetle2_ESP32S3 // Has PSRAM
+//#define CAMERA_MODEL_DFRobot_Romeo_ESP32S3 // Has PSRAM
+#include "camera_pins.h"
+
+// ===========================
+// Enter your WiFi credentials
+// ===========================
+const char* ssid = "HUAWEI Y7 Pro 2019";
```
Comment (Shehan013) [inline CameraWebServer/CameraWebServer.ino:38]: Please remove this username and password 

Use 
const char *ssid = "**********";
const char *password = "**********";

File: Msg_API_with_map_and_GPS_ALL_8266/Msg_API_with_map_and_GPS_ALL_8266.ino [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+#include <SoftwareSerial.h>
+#include <TinyGPS++.h> 
+
+#define gpsRxPin 5      //Rx pin of gps D1
+#define gpsTxPin 4      //Tx pin of gps D2
+SoftwareSerial neo6m(gpsTxPin, gpsRxPin);
+
+TinyGPSPlus gps;
+
+const char* ssid = "Galaxy S21 FE 5G 64df";
```
Comment (Shehan013) [inline Msg_API_with_map_and_GPS_ALL_8266/Msg_API_with_map_and_GPS_ALL_8266.ino:12]: Please change this

File: Msg_API_with_map_and_GPS_ALL_8266/Msg_API_with_map_and_GPS_ALL_8266.ino [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    HTTPClient http;
+    WiFiClientSecure wifiClient;
+    wifiClient.setInsecure();
+
+    http.begin(wifiClient, "https://app.notify.lk/api/v1/send");
+    http.addHeader("Content-Type", "application/x-www-form-urlencoded");
+
+    // replace user_id & api_key vakues with your new values (Current values are for Robin Hood)
+    String mapLink = generateMapLink(lat, lon);
+    String receiverNumber = "94724043919";
```
Comment (Shehan013) [inline Msg_API_with_map_and_GPS_ALL_8266/Msg_API_with_map_and_GPS_ALL_8266.ino:88]: I think it is better 94**** this way and and a //comment
further more the next line sting if it is an unique one do the same.

File: Project_code_b4_MIC/Project_code_b4_MIC.ino [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+#include <Adafruit_Sensor.h>
+#include <BH1750.h>  // Include the BH1750 library for the light sensor
+
+Adafruit_MPU6050 mpu; // MPU6050 object
+TinyGPSPlus gps; // GPS object
+HardwareSerial neo6m(1); // Use hardware serial port 1 on ESP32
+BH1750 lightMeter;  // Create an instance of the BH1750 sensor
+
+// WiFi credentials
+const char* ssid = "Mano";
```
Comment (Shehan013) [inline Project_code_b4_MIC/Project_code_b4_MIC.ino:17]: Please change

**Review by Shehan013 — CHANGES_REQUESTED** [review]

File: CameraWebServer/CameraWebServer.ino (hunk 1 above)
Comment (Kavindya-Kariyawasam) [reply CameraWebServer/CameraWebServer.ino]: Fixed the requested concerns, please review and merge.

**Review by Shehan013 — APPROVED** [review]

**Pull request merged**